mmdet/core/bbox/coder/tblr_bbox_coder.py

```diff
@@ -133,43 +135,58 @@ def tblr2bboxes(priors,
 
     Args:
         priors (Tensor): Prior boxes in point form (x0, y0, x1, y1)
-          Shape: (n,4).
+          Shape: (N,4) or (B, N, 4).
         tblr (Tensor): Coords of network output in tblr form
-          Shape: (n, 4).
+          Shape: (N, 4) or (B, N, 4).
         normalizer (Sequence[float] | float): Normalization parameter of
           encoded boxes. By list, it represents the normalization factors at
           tblr dims. By float, it is the unified normalization factor at all
           dims. Default: 4.0
         normalize_by_wh (bool): Whether the tblr coordinates have been
           normalized by the side length (wh) of prior bboxes.
-        max_shape (tuple, optional): Shape of the image. Decoded bboxes
-          exceeding which will be clamped.
+        max_shape (Sequence[int] or torch.Tensor or Sequence[
+            Sequence[int]],optional): Maximum bounds for boxes, specifies
+            (H, W, C) or (H, W). If priors shape is (B, N, 4),then the
```

Comment:
If priors shape is (B, N, 4), then the max_shape should be a Sequence[Sequence[int]] and the length of max_shape should also be B.